conjur/ssl_service.py

```diff
@@ -37,7 +44,22 @@ def __connect(cls, hostname, port):
         """
         Method for opening a socket to the Conjur server
         """
-        ctx = SSL.Context(method=SSL.TLSv1_2_METHOD)
+        # Makes the TLS version configurable through an ENV variable
+        if os.getenv('TLS_VERSION') in _tls_methods:
+            ctx = SSL.Context(method=_tls_methods[os.getenv('TLS_VERSION')])
```

Comment:
who is encharge of this env variable?  
1) recommend `tls_version = os.getenv('TLS_VERSION')` instead of reading it twice. I don't know how paranoid should I be but theoretically it could be change  after the if by an attacker (was it too paranoid?)
2) if we are not maintaining this env variable how can we be sure it is in the right format? maybe try catch incase there is a value but it is not in the right format, or ssl?

Reply:
1. updated to be variable for readability
2. not sure what you mean. it is just an environment var that can be set like `export TLS_VERSION=1.1`. What do you mean proper format?